include/swift/Runtime/Config.h

```diff
@@ -198,20 +198,13 @@ extern uintptr_t __COMPATIBILITY_LIBRARIES_CANNOT_CHECK_THE_IS_SWIFT_BIT_DIRECTL
 #define SWIFT_CC_c
 
 // SWIFT_CC(swift) is the Swift calling convention.
-// FIXME: the next comment is false.
-// Functions outside the stdlib or runtime that include this file may be built 
-// with a compiler that doesn't support swiftcall; don't define these macros
-// in that case so any incorrect usage is caught.
 #if __has_attribute(swiftcall)
 #define SWIFT_CC_swift __attribute__((swiftcall))
 #define SWIFT_CONTEXT __attribute__((swift_context))
 #define SWIFT_ERROR_RESULT __attribute__((swift_error_result))
 #define SWIFT_INDIRECT_RESULT __attribute__((swift_indirect_result))
 #else
-#define SWIFT_CC_swift
-#define SWIFT_CONTEXT
-#define SWIFT_ERROR_RESULT
-#define SWIFT_INDIRECT_RESULT
+#error "The runtime must be built with a compiler that supports swiftcall calling conventions."
 #endif
```

Comment:
I think that I would prefer something like this:

```suggestion
#if !__has_attribute(swiftcall)
#error "The runtime must be buitl with a compiler that supports the `swiftcall` calling convention."
#endif

#define SWIFT_CC_swift __attribute__((swiftcall))
#define SWIFT_CONTEXT __attribute__((swift_context))
#define SWIFT_ERROR_RESULT __attribute__((swift_error_result))
#define SWIFT_INDIRECT_RESULT __attribute__((swift_indirect_result))
```

Reply:
I wonder if we should do something per attribute that we are using?  They were introduced sequentially and a version of clang may support one but not the others I believe.

Reply:
Alright, checking for each attribute we use